src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -92,7 +92,7 @@ def _maybe_load_file(arg):
             try:
                 return _load_file(poss_file)
             except IOError:
-                logger.debug("Failed to load @'%s', assume not a file", arg)
+                logger.info("Failed to load '%s', assume not a file", arg)
```

Comment:
Mentioning the `@` is on purpose, hence I suggest keep it.

Reply:
`arg` already contains `@` (`arg.find('@')`), the original code will print duplicated `@`.

Reply:
I think a spare `@` is not necessary.